frontend/src/types.ts

```diff
@@ -75,6 +84,12 @@ export const iconMap: IconMap = {
   [NIP_47_NOTIFICATIONS_PERMISSION]: Bell,
 };
 
+export const scopeGroupIconMap: IconMap = {
+  [SCOPE_GROUP_SEND_RECEIVE]: ArrowDownUp,
```

Comment:
The way you implemented scope groups has no type safety. Can you add a `type ScopeGroup = "send_receive" | "only_receive" | "custom"`

Reply:
Ah, I see you have `ScopeGroupType` but it's not always used. Sometimes you just use strings